app/src/main/java/com/example/campusguide/calendar/Calendar.kt

```diff
@@ -76,17 +79,11 @@ class Calendar constructor(val activity: MapsActivity, userEmail: String) {
         return calendars
```

Comment:
Should this be a deep copy?

Reply:
> addressed in inline comment but i'll respond to you here as well: the permissions are checked elsewhere. the suppression is purely so that androidstudio will stop harassing us about it in every class that uses the permissions we've already checked for